mise/tasks/app/upload-ios.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+# mise description="Creates Tuist iOS app .ipa archive and uploads it to App Store Connect."
+
+set -eo pipefail
+
+# Run the bundle script first
+echo "Building iOS app bundle..."
+$MISE_PROJECT_ROOT/mise/tasks/app/bundle-ios.sh
```

Comment:
You don't need the variable because the default working directory is `$MISE_PROJECT_ROOT`:

```suggestion
mise/tasks/app/bundle-ios.sh
```